tasks-11-21-22-23-24/Dao/BrokerNodeDAO.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Dao;
+
+require_once("Lib/Database.php");
+require_once("Model/BrokerNodeModel.php");
+
+use Lib\Database;
+use Model\BrokerNodeModel;
+
+class BrokerNodeDAO extends Database {
+	
+	function __construct() {
+		parent::__construct();
+	}
+	
+	public function insertNode($data) {
+		$this->bindMore(array("ip_address" => $data["ipAddress"]));
+		$insert = $this->query("INSERT INTO brokernode(ip_address) VALUES(:ip_address)");
+		return $insert;
+	}
+	
+	public function verifyRegisteredBrokerNode($ipAddress) {
```

Comment:
Same here

Reply:
Hy @rfornea Can you clarify me please. These 2 functions or the classes needs to go to Hooknode repo?